addons/medical_engine/functions/fnc_handleDamage.sqf

```diff
@@ -211,9 +211,12 @@ if (
     // todo: no way to detect if stationary and another vehicle hits you
 ) exitWith {
     private _damageSelectionArray = [
-        HITPOINT_INDEX_HEAD, 1, HITPOINT_INDEX_BODY, 1, HITPOINT_INDEX_LARM, 1, 
+        HITPOINT_INDEX_HEAD, 1, HITPOINT_INDEX_BODY, 1, HITPOINT_INDEX_LARM, 1,
         HITPOINT_INDEX_RARM, 1, HITPOINT_INDEX_LLEG, 1, HITPOINT_INDEX_RLEG, 1
     ];
+    if (!EGVAR(medical,enableVehicleCrashes)) then {
+        _newDamage = 0;
```

Comment:
This still triggers the `QEGVAR(medical,woundReceived)` event. When vehicle crashes are disabled, I think we wouldn't want "vehiclecrash" wound received events.

Reply:
Fair enough. Forgot to account for that.